package.json

```diff
@@ -37,15 +48,19 @@
     "doc": "node utils/doclint/cli.js",
     "generate-api-docs-for-testing": "commonmark docs/api.md > docs/api.html",
     "clean-lib": "rimraf lib",
-    "build": "npm run tsc && npm run generate-d-ts",
-    "tsc": "npm run clean-lib && tsc --version && npm run tsc-cjs && npm run tsc-esm",
+    "build": "npm run tsc && npm run generate-d-ts && npm run generate-pkg-json",
```

Comment:
Let's include everything generated in https://github.com/puppeteer/puppeteer/blob/main/.versionrc.js#L23 I don't think we should generate during build, rather we should generate when the version is changed.